kong/runloop/handler.lua

```diff
@@ -934,6 +941,58 @@ return {
       -- ngx.req.start_time() is kept in seconds with millisecond resolution.
       ctx.KONG_PROXY_LATENCY   = now - ngx.req.start_time() * 1000
       ctx.KONG_PROXIED         = true
+
+      -- clear hop-by-hop request headers:
```

Comment:
I think these additions should go above the line with `now = get_now()` (i.e., time doing this should be included in the KONG_ACCESS_TIME calculation)

Reply:
Changed the order.